script/BaseScript.s.sol

```diff
@@ -169,62 +168,25 @@ contract BaseScript is BaseData, Utils {
         );
     }
 
-    function _computeCreate3Address(bytes32 _salt, address _deployer) internal pure returns (address) {
-        bytes memory proxyByteCode = hex"67363d3d37363d34f03d5260086018f3";
-
-        bytes32 proxyByteCodeHash = keccak256(proxyByteCode);
-        address proxy =
-            keccak256(abi.encodePacked(bytes1(0xFF), _deployer, _salt, proxyByteCodeHash)).fromLast20Bytes();
-
-        return keccak256(abi.encodePacked(hex"d694", proxy, hex"01")).fromLast20Bytes();
-    }
-
     function _loadPredictions() internal {
-        {
-            bytes32 salt = createImmutableMultiChainDeployerSalt(msg.sender);
-
-            address predictedAddress =
-                vm.computeCreate2Address(salt, keccak256(type(ImmutableMultiChainDeployer).creationCode));
-            predictions.l2MultiChainDeployer = predictedAddress;
-        }
-
         {
             bytes32 proxySalt = createL1YnOFTAdapterUpgradeableProxySalt(msg.sender);
-            bytes32 implementationSalt = createL1YnOFTAdapterUpgradeableSalt(msg.sender);
 
-            bytes memory implBytecode = bytes.concat(
-                type(L1YnOFTAdapterUpgradeable).creationCode,
-                abi.encode(baseInput.l1ERC20Address, getData(baseInput.l1ChainId).LZ_ENDPOINT)
-            );
-
-            address predictedImplementation = vm.computeCreate2Address(implementationSalt, keccak256(implBytecode));
-
-            bytes memory initializeData =
-                abi.encodeWithSelector(L1YnOFTAdapterUpgradeable.initialize.selector, msg.sender);
-
-            bytes32 timelockSalt = createL1YnOFTAdapterTimelockSalt(msg.sender);
-
-            address predictedTimelock = _predictTimelockController(timelockSalt, baseInput.l1ChainId);
-
-            bytes memory proxyBytecode = bytes.concat(
-                type(TransparentUpgradeableProxy).creationCode,
-                abi.encode(predictedImplementation, predictedTimelock, initializeData)
-            );
-
-            address predictedAddress = vm.computeCreate2Address(proxySalt, keccak256(proxyBytecode));
```

Comment:
out of curiosity why does this function do all of this with create2  instead of doing the thing now on the right?

Was it because you weren't aware at the time they're requivalent or there was a better reason for this?

Reply:
because the l1Adapter is deployed without the create3 factory.  Just using create 2.  we can change this if you want but to keep it consistent with the previous deployments we left the l1 deploying with just create2.  this makes it so the l1 adapter will have one address and the l2s will have another.